php-fpm/docker-php-ext-apcu-cli.ini

```diff
@@ -1 +1,2 @@
 apc.enable_cli = On
+apc.shm_size = 256M
```

Comment:
Should we somewhere document that the usage now required this to have a configuration?

Reply:
this is not strictly required. It just turns out that current performance tests process too much data for the former limit (not sure what it was by default, maybe 32 MB?), and as a result some data were expunged during the tests, so the result numbers were inaccurate. you can see that in test durations, after increasing shm size the most data intensive tests take longer